### ISSUE TITLE
Add up alert rule

### ISSUE THREAD

**Issue by dstathis** [canonical/prometheus-scrape-target-k8s-operator#42]
### Enhancement Proposal

It would be easy and a free win to add an "up" alert rule for each target.

**Comment by sed-i** [comment]
@MichaelThamm would [generic alert rules](https://github.com/canonical/prometheus-k8s-operator/pull/660) be generated for e.g. avalanche in the following deployment:
```mermaid
graph LR
avalanche -.-|"(no relation!)"| scrape-target ---|"scrape job for avalanche"| prometheus
```

**Comment by MichaelThamm** [comment]
No, because scrape-target is not using `MetricsEndpointProvider` or `PrometheusRemoteWriteConsumer` which is used to inject the generic alert rules.

We would need to implement either to have `scrape-target` follow the generic alert rule workflow.
